examples/phoenix_app/lib/phoenix_app_web/router.ex

```diff
@@ -16,8 +16,14 @@ defmodule PhoenixAppWeb.Router do
     get("/swaggerui", OpenApiSpex.Plug.SwaggerUI,
       path: "/api/openapi",
       default_model_expand_depth: 3,
-      display_operation_id: true
+      display_operation_id: true,
+      oauth2_redirect_url: "http://localhost:4000/swaggerui/oauth2-redirect.html",
```

Comment:
TODO: avoid hard-coding the host/port here.

Reply:
One option is to set the `config_url` option, and provide a route to fetch the configuration dynamically.